apps/nfw-react/src/app/project/store/data/order/order.reducers.js

```diff
@@ -0,0 +1,91 @@
+import * as orderAT from './order.action-types';
+import { v4 } from 'uuid';
+
+const initialState = {
+  pizza: [],
+  sauce: [],
+  total: 0,
+};
+
+export default function orderReducer(state = initialState, action) {
+  switch (action.type) {
+    case orderAT.ADD_PIZZA: {
+      return Object.assign({}, state, {
+        pizza: state.pizza.concat({
+          id: action.payload.id,
+          ingredients: action.payload.ingredients,
+        }),
+        total: state.total + action.payload.price,
```

Comment:
(should) I'd not hold total in state -> I'd move that logic into the selector and then before order invoke it + use in card cmp to display total based on order state (pizzas and sauces)